crates/consensus/src/receipt/any.rs

```diff
@@ -86,7 +87,7 @@ impl<T> AnyReceiptEnvelope<T> {
     }
 }
 
-impl<T> TxReceipt<T> for AnyReceiptEnvelope<T> {
+impl<T: Clone + fmt::Debug + PartialEq + Eq + Send + Sync> TxReceipt<T> for AnyReceiptEnvelope<T> {
```

Comment:
this shouldn't be squeezed into the `<>` looks bad, needs where clause

Reply:
fixed